src/status_im/ui/components/common/common.cljs

```diff
@@ -86,7 +86,7 @@
     [react/text (cond-> {:style (styles/counter-label size)}
                   accessibility-label
                   (assoc :accessibility-label accessibility-label))
-     value]]))
+     (if (<= value 99) value "99+")]]))
```

Comment:
Can you use i18n here? And store this in `en.cljs` ?

Reply:
fixed.